osbs/core.py

```diff
@@ -43,14 +43,17 @@
 logger = logging.getLogger(__name__)
 
 
-def check_response(response):
+def check_response(response, log_errors=True):
     if response.status_code not in (httplib.OK, httplib.CREATED):
         if hasattr(response, 'content'):
             content = response.content
         else:
             content = b''.join(response.iter_lines())
 
-        logger.error("[%d] %s", response.status_code, content)
```

Comment:
How about using `logger.log`[0]?
https://docs.python.org/2/library/logging.html#logging.Logger.log

In that case `check_response` could take the log level to use, defaulting to `logging.ERROR`.